File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.cs
using System;

public class Solution {
    public int solution(string binomial) {
        string[] bin = binomial.Split(' ');
        int answer = 0;
        switch(bin[1]){
            case "+":
               answer = Int32.Parse(bin[0]) + Int32.Parse(bin[2]);
                break;
            case "-":
                answer = Int32.Parse(bin[0]) - Int32.Parse(bin[2]);
                break;
            case "*":
                answer = Int32.Parse(bin[0]) * Int32.Parse(bin[2]);
                break;         
        }
        return answer;
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string binomial) 
{
    vector<string> binmVec;
    char space = ' ';
    string temp = "";
    for(char &ch : binomial)
    {
        if(ch == ' ')
        {
            binmVec.push_back(temp);
            temp = "";
        }
        else
        {
            temp += ch;
        }
    }
    binmVec.push_back(temp);
    
    
    int answer = 0;
    string strop = binmVec[1];
    char op = strop[0];
    switch(op)
    {
        case '+':
            answer = stoi(binmVec[0]) + stoi(binmVec[2]);
            break;
        case '-':
            answer = stoi(binmVec[0]) - stoi(binmVec[2]);
            break;
        case '*':
            answer = stoi(binmVec[0]) * stoi(binmVec[2]);
            break;
    }
    return answer;
}